<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-CVKJJ5N" failures="1" tests="9" name="TestListeners" time="0.138" errors="0" timestamp="2022-03-29T16:41:42 SAST">
  <testcase classname="secondTestPackage.secondClassSecondPackage" name="firstTestSecondclassSecondPackage" time="0.009"/>
  <testcase classname="secondTestPackage.secondClassSecondPackage" name="fourthTestSecondClassSecondPackage" time="0.002"/>
  <testcase classname="secondTestPackage.secondClassSecondPackage" name="secondTestSecondClassSecondPackage" time="0.003"/>
  <testcase classname="secondTestPackage.secondClassSecondPackage" name="thirdTestSecondClassSecondPackage" time="0.004"/>
  <testcase classname="secondTestPackage.firstClassSecondPackage" name="failedTest" time="0.005">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at secondTestPackage.firstClassSecondPackage.failedTest(firstClassSecondPackage.java:33)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- failedTest -->
  <testcase classname="secondTestPackage.firstClassSecondPackage" name="firstTestFirstClassSecondPackage" time="0.003"/>
  <testcase classname="secondTestPackage.firstClassSecondPackage" name="fourthTestSecondPackage" time="0.002"/>
  <testcase classname="secondTestPackage.firstClassSecondPackage" name="skippedTest" time="0.003">
    <skipped/>
  </testcase> <!-- skippedTest -->
  <testcase classname="secondTestPackage.firstClassSecondPackage" name="thirdTestFirstClassSecondPackage" time="0.007"/>
  <testcase classname="secondTestPackage.firstClassSecondPackage" name="secondTestFirstClassSecondPackage" time="0.0">
    <ignored/>
  </testcase> <!-- secondTestFirstClassSecondPackage -->
</testsuite> <!-- TestListeners -->
